src/ORM/Connect/MySQLiConnector.php

```diff
@@ -121,6 +121,13 @@ public function connect($parameters, $selectDB = false)
         if (!empty($collation)) {
```

Comment:
Do we still need this query? I seems like a double-up, although I'm not sure how to test for it.